stage0/run.go

```diff
@@ -452,6 +457,8 @@ func Run(cfg RunConfig, dir string, dataDir string) {
 		log.Fatalf("error clearing FD_CLOEXEC on lock fd")
 	}
 
+	tpmEvent := fmt.Sprintf("rkt: Rootfs: %s Manifest: %s Stage 1 args: %s", cfg.CommonConfig.RootHash, cfg.CommonConfig.ManifestData, strings.Join(args, " "))
+	tpm.Extend(tpmEvent)
```

Comment:
Error handling? What's the desired/expected behaviour if rkt is compiled with tpm but a) the system doesn't actually have it, or b) the operation otherwise fails?